RFCs/0009-Editors-API-RevisionUpdate.md

```diff
@@ -0,0 +1,350 @@
+# 0009 Editors API Revision Update
+
+## Summary
+This note outlines the proposed changes for the JSON protocol between an Editor/IDE
+and the test platform.
+
+## Overview of changes
+
+### Communication Message
+Protocol related communication messages between Editor and Test Platform will have the following structure:
+
+#### API Payload
```

Comment:
Mention that in v2: this message will transfer once at the beginning. After negotiation, all other messages follow the versioned message.